quickstarts/microsoft.machinelearningservices/machine-learning-end-to-end-secure/README.md

```diff
@@ -0,0 +1,79 @@
+
+This set of Bicep templates demonstrates how to set up Azure Machine Learning end-to-end in a secure set up.
+
+This reference implementation includes the Workspace, a CPU compute cluster, a GPU compute cluster, compute instance and attached private AKS cluster. It  includes the configuration of associated resources including Azure Key Vault, Azure Storage, Azure Application Insights and Azure Container Registry in a network-isolated setup.
+
+## Resources
+
+| Provider and type | Description |
+| - | - |
+| `Microsoft.Resources/resourceGroups` | The resource group all resources get deployed into |
+| `Microsoft.Insights/components` | An Azure Application Insights instance associated to the Azure Machine Learning workspace |
+| `Microsoft.KeyVault/vaults` | An Azure Key Vault instance associated to the Azure Machine Learning workspace |
+| `Microsoft.Storage/storageAccounts` | An Azure Storage instance associated to the Azure Machine Learning workspace |
+| `Microsoft.ContainerRegistry/registries` | An Azure Container Registry instance associated to the Azure Machine Learning workspace |
+| `Microsoft.MachineLearningServices/workspaces` | An Azure Machine Learning workspace instance |
+| `Microsoft.MachineLearningServices workspaces/computes` | Azure Machine Learning workspace compute types: cluster and compute instance |
+| `Microsoft.Network/privateDnsZones` | Private DNS zones for Azure Machine Learning and the dependent resources |
+| `Microsoft.Network/networkSecurityGroups` | A Network Security Group pre-configured for use with Azure Machine Learning |
+| `Microsoft.ContainerService/managedClusters` | An Azure Kubernetes Services cluster for inferencing |
+| `Microsoft.Compute/virtualMachines` | A Data Science Virtual Machine `jumpbox` to access the workspace over the private link endpoint |
+| `Microsoft.Network/virtualNetworks` | A virtual network to deploy all resources in |
+
+## Variables
+
+| Name | Description | Default |
+|-|-|-|
+| location | Specifies the Azure region for all resources | resourceGroup().location |
+| prefix | Specifies the prefix for all resources created in this deployment | - |
+| tags | Any tags you want to apply to all resources | {} |
+| vnetAddressPrefix | Specifies the address prefix of the virtual network. | '192.168.0.0/16' |
+| trainingSubnetPrefix | Specifies the address prefix of the training subnet. | '192.168.0.0/24' |
+| scoringSubnetPrefix | Specifies the address prefix of the scoring subnet. | '192.168.1.0/24' |
+| azureBastionSubnetPrefix | Specifies the address prefix of the azure bastion subnet. | '192.168.250.0/27' |
+| dsvmJumpboxUsername | Admin username for the Data Science Virtual Machine (jumpbox) | - |
+| dsvmJumpboxPassword | Admin password for the Data Science Virtual Machine (jumpbox) | - |
+
+## Usage
```

Comment:
Could you please align with Azure/Terraform repos so we can keep the docs consistent?

Reply:
Not likely, as we don't have anything in place that will update 1K samples if something changes - and since all the samples follow the same process for deployment it's redundant.

Reply:
Fair enough